compose/project.py

```diff
@@ -1,16 +1,20 @@
 from __future__ import unicode_literals
 from __future__ import absolute_import
-import logging
 from functools import reduce
+import logging
+import os
 
+import concurrent.futures
 from docker.errors import APIError
 
 from .config import get_service_name_from_net, ConfigurationError
-from .const import LABEL_PROJECT, LABEL_SERVICE, LABEL_ONE_OFF, DEFAULT_TIMEOUT
+from .const import (DEFAULT_MAX_WORKERS, DEFAULT_TIMEOUT, LABEL_PROJECT,
+                    LABEL_SERVICE, LABEL_ONE_OFF)
```

Comment:
You don't normally split them out individually onto each line, PEP8 recommends a few options for dealing with long imports, hence me putting them in brackets. 

What I could have done to be more helpful was in a separate commit, re-order them. Then another one adding in the import, I conflated the two which feels more disruptive. I'll keep this in mind for next time.